index.md

```diff
@@ -22,26 +22,26 @@ GhostText is a browser extension that connects to your editor via its own extens
 
 {% include editors.html %}
 
-Most editor extensions are by third parties. You can create more extensions for your editor! Refer to the [protocol](https://github.com/fregante/GhostText/blob/main/PROTOCOL.md) file.
+Most editor extensions are authored by third parties. You can create more extensions for your favorite editor! Refer to [the protocol document](https://github.com/fregante/GhostText/blob/main/PROTOCOL.md).
 
 ## Usage
 
-Once you install both extensions, you can activate GhostText in the current browser tab by clicking its icon in the toolbar or by using a [keyboard shortcut](#keyboard-shortcuts). The editor will need to be already open.
+Once you have installed both browser and editor extensions, you can activate GhostText in the current browser tab by clicking its icon in the toolbar or by using a [keyboard shortcut](#keyboard-shortcuts). The editor needs to be launched first.
 
-Notice: in some editors you’ll need to run the _Enable GhostText_ command to start the server. Refer to the documentation of your editor’s GhostText extension. Sublime Text doesn’t need this step.
+Notice: in some editors you’ll need to run the _Enable GhostText_ command to start the server. Refer to the documentation of each extension. Sublime Text doesn’t need this step.
 
 ## How it works
 
 GhostText is split in two parts:
 
-1. a HTTP and WebSocket server in the text editor
-2. a client in the browser
+• A HTTP and WebSocket server in the text editor  
```

Comment:
These should be regular dashes. Also there's trailing space

Reply:
This is intentional because [these styles](https://github.com/fregante/ghosttext.fregante.com/blob/main/style.css#L111-L115) doesn't look right here. But it's seems they are only needed in the troubleshooting page so ideally they should be scoped to that page.

Reply:
That might why I was using 1.2. in the first place.  😃 Using • is a downgrade in semantic and accessibility. 

I'm not sure if there's an easy way to add a stylesheet just on that page. Maybe it can be done by wrapping that troubleshooting list in a `<div class=something>` in order to be targetable.